estate/models/estate_property_offer.py

```diff
@@ -28,3 +34,19 @@ def _compute_date_deadline(self):
         for record in self:
             start_date = record.create_date if record.create_date else fields.Date.today()
             record.date_deadline = fields.Date.add(start_date, days=record.validity)
+
+    def action_accept_offer(self):
+        self.ensure_one()
+
+        if self.property_id.buyer_id:
+            raise UserError(_("There is an offer accepted already for this property."))
+        self.status = "accepted"
+        self.property_id.selling_price = self.price
+        self.property_id.buyer_id = self.partner_id
+        return True
+
+    def action_refuse_offer(self):
+        self.ensure_one()
+
+        self.status = "refused"
```

Comment:
Return selling_price and buyer_id values so that another offer can be accepted.

```suggestion
        self.status = "refused"
        self.property_id.selling_price = 0
        self.property_id.buyer_id = False
```

Reply:
In the case that I first accept an offer and then refuse the others, finally the `selling_price` value will be `0` and buyer as `False`.